src/EditorFeatures/CSharpTest/SignatureHelp/InvocationExpressionSignatureHelpProviderTests.cs

```diff
@@ -2022,5 +2022,77 @@ void M()
 
             await TestAsync(markup, new[] { new SignatureHelpTestItem("void List<int>.Add(int item)") });
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.SignatureHelp)]
+        public async Task TypingTupleDoesNotDismiss1()
+        {
+            var markup = @"
+class C
+{
+    int Foo(object x)
+    {
+        [|Foo(($$)|];
+    }
+}";
+
+            var expectedOrderedItems = new List<SignatureHelpTestItem>();
+            expectedOrderedItems.Add(new SignatureHelpTestItem("int C.Foo(object x)", currentParameterIndex: 0));
+
+            await TestAsync(markup, expectedOrderedItems, usePreviousCharAsTrigger: true);
```

Comment:
In this case, dismissal happens because the user types a '(' or ',' and the system asks the provider to trigger on that typed char. We used to dismiss because we wouldn't trigger on the start of a tuple. I'll rename the tests to indicate this.